src/components/Input/index.tsx

```diff
@@ -0,0 +1,66 @@
+import { useState } from 'react';
+import styled from 'styled-components';
+import InputWithButton from '@/components/Input/InputWithButton';
+
+const Input = ({ children, className }: { className?: string; children?: JSX.Element }) => {
+  const [value, setValue] = useState('');
+  const id = 'temp';
+
+  return (
+    <Wrapper className={className}>
+      <label htmlFor={id}>제목</label>
+      <div className='input'>
+        <input
+          type='text'
+          name=''
+          id={id}
+          value={value}
+          onChange={(e) => {
+            setValue(e.target.value);
+          }}
+        />
+        {children}
+      </div>
+      <span className='error'>에러메세지</span>
+    </Wrapper>
+  );
+};
+
+Input.deafultProps = {
+  className: '',
+};
+
+export default Input;
+export { InputWithButton };
+
+const Wrapper = styled.div`
```

Comment:
```suggestion
const Wrapper = styled.div<{props? : any}>`
```
이런식으로 스타일드 컴포넌트에 props를 전달해서 처리할 수 있을까요?

Reply:
넵 맞습니다! 다만 패딩이나 마진값 추가만을 위한 의도로 className을 추가했습니다. 유의미한 props를 정의하기 애매해서요